care/facility/migrations/0469_remove_scheduleexception_datetime_range_and_more.py

```
@@ -0,0 +1,61 @@
+            model_name='scheduleexception',
+            name='reason',
+            field=models.TextField(blank=True, null=True),
+        ),
+        migrations.AddField(
+            model_name='scheduleexception',
+            name='start_time',
+            field=models.TimeField(default='00:00'),
+            preserve_default=False,
+        ),
+        migrations.AddField(
+            model_name='scheduleexception',
```

Ruff

Single quotes found but double quotes preferred

Replace single quotes with double quotes

(Q000)